packages/frontend/editor-ui/src/composables/usePushConnection/handlers/nodeExecuteAfterData.test.ts

```diff
@@ -0,0 +1,99 @@
+import { createTestingPinia } from '@pinia/testing';
+import { setActivePinia } from 'pinia';
+import { nodeExecuteAfterData } from './nodeExecuteAfterData';
+import { useWorkflowsStore } from '@/stores/workflows.store';
+import { mockedStore } from '@/__tests__/utils';
+import { TRIMMED_TASK_DATA_CONNECTIONS_KEY } from 'n8n-workflow';
+import type { NodeExecuteAfterData } from '@n8n/api-types/push/execution';
+import type { IExecutionResponse } from '@/Interface';
+import { createTestWorkflow } from '@/__tests__/mocks';
+
+describe('nodeExecuteAfterData', () => {
+	beforeEach(() => {
+		const pinia = createTestingPinia({
+			stubActions: true,
+		});
+		setActivePinia(pinia);
+	});
+
+	it('should call clearNodeExecutionData if data has trimmed items', async () => {
+		const workflowsStore = mockedStore(useWorkflowsStore);
+
+		workflowsStore.workflowExecutionData = {
```

Comment:
Nitpick, but we can use `createTestWorkflowExecutionResponse` `createTestTaskData` so we don't have to hardcode all fields and can remove type assertions

Reply:
Thanks, didn't realize we have that helper.